requirements.txt

```diff
@@ -6,4 +6,6 @@ qiskit-terra
 qiskit-aer
 pytket>1.0,<2.0
 setproctitle
-rich
\ No newline at end of file
+rich
+configparser
```

Comment:
What is this used for? `ini` file parsing should be in python's stdlib: https://docs.python.org/3/library/configparser.html so we shouldn't need to add a dependency to do it.

Reply:
You are right, I'll fix that now.